src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,271 @@
 import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import './NewMovie.scss';
+
+// eslint-disable-next-line
+const regexp = /^((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w-_]*)?\??(?:[-+=&;%@.\w_]*)#?(?:[.!/\\\w]*))?)$/;
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+    hasTitleError: false,
+    hasImgUrlError: false,
+    hasImdbUrlError: false,
+    hasImdbIdError: false,
+    buttonDisabled: false,
+  };
+
+  handleChange = (event) => {
+    const { name } = event.target;
+    const { value } = event.target;
```

Comment:
можно достать все за один раз
```suggestion
    const { name, value } = event.target;
```